build.gradle

```diff
@@ -16,6 +16,7 @@ buildscript {
     classpath "org.asciidoctor:asciidoctor-gradle-plugin:1.6.0"
     classpath 'org.ajoberstar.grgit:grgit-core:3.0.0'
     classpath 'org.ajoberstar:gradle-git-publish:2.0.0'
+    classpath "gradle.plugin.org.jetbrains.gradle.plugin.idea-ext:gradle-idea-ext:0.5"
```

Comment:
Doesnt using `https://plugins.gradle.org/plugin/net.ltgt.apt-idea` fix this for you in the IDE automatically?

Reply:
I seem that `net.ltgt.apt-idea` is no longer maintained.

ref: https://github.com/tbroyer/gradle-apt-plugin/blob/master/README.md

Reply:
And we need this plugin [here](https://github.com/springfox/springfox/pull/2971/files#diff-ed433482e37889a2349d935928977bb3R9-R15)

Reply:
@dilipkrish Could you review again?